mlir/tools/rocmlir-lib/CMakeLists.txt

```diff
@@ -171,7 +189,11 @@ if(BUILD_FAT_LIBROCKCOMPILER)
   set(extra_include_dir "")
 
   if ((NOT LLVM_INSTALL_TOOLCHAIN_ONLY) AND EXPORT_ALL_HEADERS)
-    install(DIRECTORY
+  rocm_package_setup_component(${LIBRARY_NAME}-headers
+        LIBRARY_NAME ${LIBRARY_NAME}-headers
+        PACKAGE_NAME ${PACKAGE_NAME}
+  )
```

Comment:
Why is this `${LIBRARY_NAME}-headers` component being used? I would expect these and the header file above to simply go into the `devel` component.